Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN apk add --update python build-base && npm install && apk del python build-ba
 EXPOSE 8890
 
 # Default environment variables
-ENV ZOOKEEPER_HOST zookeeper
+ENV ZOOKEEPER_HOST zookeeper:9092
```

Comment:
`9092` is not a zookeeper port. Are you trying to pass a kafka node address in the value `ZOOKEEPER_HOST`? You should probably rename the env variable.